externals/kyuubi-spark-sql-engine/src/main/resources/python/execute_python.py

```diff
@@ -15,28 +15,36 @@
 # limitations under the License.
 #
 
-from glob import glob
 import ast
-import sys
 import io
 import json
-import traceback
-import re
+import kyuubi_util
 import os
+import re
+import sys
+import traceback
+from glob import glob
+
+if sys.version >= '3':
+    unicode = str
+else:
+    sys.exit('Python < 3 is unsupported.')
 
 # ast api is changed after python 3.8, see https://github.com/ipython/ipython/pull/11593
-if sys.version_info > (3,8):
-  from ast import Module
-else :
-  # mock the new API, ignore second argument
-  # see https://github.com/ipython/ipython/issues/11590
-  from ast import Module as OriginalModule
-  Module = lambda nodelist, type_ignores: OriginalModule(nodelist)
+if sys.version_info > (3, 8):
```

Comment:
maybe `>=`?

Reply:
OK, changed to `>=`.
The original patch is from `https://github.com/apache/incubator-livy/pull/314`. The title claims support for 3.8+, but the code only check version greater than 3.8 .